help/destinations/catalog/crm/microsoft-dynamics-365.md

```diff
@@ -0,0 +1,218 @@
+        |XDM Profile Schema|[!DNL Dynamics 365] Instance|
+        |---|---|
+        |`person.name.firstName`|`FirstName`|
+        |`person.name.lastName`|`LastName`|
+        |`personalEmail.address`|`Email`|
+
+    * An example using these mappings is shown below:
+    ![Platform UI screenshot example showing Target mappings.](../../assets/catalog/crm/microsoft-dynamics-365/mappings.png)
+
+### Schedule segment export and example {#schedule-segment-export-example}
+
+When performing the [Schedule segment export](../../ui/activate/activate-segment-streaming-destinations.md) step you must manually map Platform segments to the custom field attribute in [!DNL Dynamics 365].
+
+To do this, select each segment, then enter the corresponding custom field attribute from [!DNL Dynamics 365] in the [!UICONTROL Mapping ID] field.
+
+> [!IMPORTANT]
+>
+> The value used for the [!UICONTROL Mapping ID] should exactly match the name of the custom field attribute created within [!DNL Dynamics 365]. See [[!DNL Dynamics 365] documentation](https://docs.microsoft.com/en-us/dynamics365/customerengagement/on-premises/customize/create-edit-fields?view=op-9-1) if you need guidance on finding your custom field attributes.
+
+An example is shown below:
+![Platform UI screenshot example showing Schedule segment export.](../../assets/catalog/crm/microsoft-dynamics-365/schedule-segment-export.png)
+
+## Validate data export {#exported-data}
+
+To validate that you have correctly set up the destination, follow the steps below:
+
+1. Select **[!UICONTROL Destinations]** > **[!UICONTROL Browse]** to navigate to the list of destinations.
+![Platform UI screenshot showing Browse Destinations.](../../assets/catalog/crm/microsoft-dynamics-365/browse-destinations.png)
+
+1. Select the destination and validate that the status is **[!UICONTROL enabled]**.
+![Platform UI screenshot showing Destinations Dataflow Run.](../../assets/catalog/crm/microsoft-dynamics-365/destination-dataflow-run.png)
+
+1. Switch to the **[!DNL Activation data]** tab, then select a segment name.
+![Platform UI screenshot example showing Destinations Activation Data.](../../assets/catalog/crm/microsoft-dynamics-365/destinations-activation-data.png)
+
+1. Monitor the segment summary and ensure that the count of profiles corresponds to the count created within the segment.
+![Platform UI screenshot example showing Segment.](../../assets/catalog/crm/microsoft-dynamics-365/segment.png)
+
+1. Log in to the Dynamics 365 website, then navigate to the **[!DNL Apps]** > **[!DNL Contacts]** page and check if the profiles from the segment have been added. You will also notice the segment status from Experience Platform has been updated against the corresponding custom field attribute that was provided in the **Mapping ID** field during the Platform **[!UICONTROL Activate destination]** > **[!UICONTROL Schedule segment export]** step.
```

Comment:
Line 187: I think I already mentioned this in my previous comments. There is no **[!UICONTROL Activate destination]** UI element. Please rephrase this section as below:

"You can see that each segment status from [!DNL Dynamics 365] was updated with the corresponding segment status from Platform, based on the custom field attribute that you entered in the [!UICONTROL Mapping ID] field, during the segment scheduling step."

Reply:
Had responded with the substantiation and requested for your guidance https://github.com/AdobeDocs/experience-platform.en/pull/90#discussion_r942611574 replicated below.

We are referring to the Platform [step](https://github.com/AdobeDocs/experience-platform.en/blob/1b3618b96a31bbada9f74e9895b9d546e109c52a/help/destinations/assets/catalog/crm/microsoft-dynamics-365/schedule-segment-export.png). This is to emphasize the corroboration between what was provided in the Platform and what is visible in Dynamics 365 contacts page. 

Have prefixed the word Platform before **[!UICONTROL Activate destination]**.

Any suggestions on what rephrasing is needed ?

Reply:
We only use [!UICONTROL] for actual UI elements that are present in the Experience Platform UI (buttons, fields).
Please see the detailed information in our authoring guide on when to use these tags https://experienceleague.adobe.com/docs/authoring-guide-exl/using/authoring/localization/localize.html?lang=en.

In this section, there is no _Activate destination_ button anywhere in the UI. The segment schedule step is already clearly documented, so you can link to that section.

As far as rephrasing suggestions go, please see my comment above. I have updated it with a clearer explanation.